src/rgw/rgw_process.cc

```diff
@@ -466,16 +466,16 @@ int process_request(const RGWProcessEnv& penv,
     rgw_log_op(rest, s, op, penv.olog.get());
   }
 
-  if (op) {
-    std::ignore = rgw::bucketlogging::log_record(driver, 
+  if (op && op->get_type() != RGW_OP_COMPLETE_MULTIPART) {
```

Comment:
process_request() should be generic, so special cases for op types here aren't ideal. if we can't come up with a better abstraction to handle this, please at least add a comment why we're skipping for CompleteMultipart

Reply:
will use the `always_log()` API that I'm adding in the other PR